index.js

```diff
@@ -330,6 +330,36 @@ walletRouter.post('/melt-tokens', (req, res) => {
   }
 });
 
+/**
+ * GET request to get utxo details
+ * For the docs, see api-docs.js
+ */
+walletRouter.get('/utxo-details', (req, res) => {
+  const wallet = req.wallet;
+  const options = req.query || {};
+  const utxoDetails = wallet.getUtxos(options);
+  res.send(utxoDetails);
+});
+
+/**
+ * POST request to consolidate utxos
+ * For the docs, see api-docs.js
+ */
+walletRouter.post('/utxo-consolidation', async (req, res) => {
+  try {
+    const wallet = req.wallet;
+    const { destination_address, ...options } = req.body || {};
```

Comment:
I feel it is risky to just pass the `options` to the `consolidateUtxos()` method. It is not a problem today but it can be a problem in future versions of the lib. Let's say that, in the future, we add a new option to the `consolidateUtxos()` method, and this option shouldn't be exposed to the headless wallet. In this case, it would be exposed and we probably wouldn't notice until a bug is reported.

In other words, I feel we should only pass the valid options to the `consolidateUtxos()` methods. What do you think?

Reply:
You are right. To allow only valid options I added `express-validator` to the middlewares chain, it will return validation errors to invalid or missing `body parameters` too (see it [here](https://github.com/HathorNetwork/hathor-wallet-headless/pull/48/commits/d0a2f74c283b31ca6d01e3f1ac9a220bd78ac407#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R355)). To me it seems to solve two problems at once, query/body properties without validation and unknown property being passed to `consolidateUtxos`. Do you like the idea?

Reply:
Seems good to me. Thanks!